typewriter/fixes/tests/test_util.py

```diff
@@ -49,7 +49,16 @@ def test(self):
                          ({"a.x": [('b', 'b')]}, "from a.x import b"),
                          ({"a.x": [('b', 'c')]}, "from a.x import b as c"),
                          ({"a": [('b', 'a.b')], "x": [('c', 'x.c')], "y": [('d', 'y.d')]},
-                          "import a.b, x.c, y.d"))
+                          "import a.b, x.c, y.d"),
+                         ({"typing": [('Any', 'Any'), ('Callable', 'Callable'),
+                                      ('Deque', 'Deque'), ('Generic', 'Generic'),
+                                      ('Iterable', 'Iterable'), ('Iterator', 'Iterator'),
+                                      ('List', 'List'), ('Optional', 'Optional'),
+                                      ('Tuple', 'Tuple'), ('TypeVar', 'TypeVar'),
+                                      ('Union', 'Union'), ('overload', 'overload')]},
+                          """from typing import (Any, Callable, Deque, Generic, Iterable, Iterator, List,
+                                                 Optional, Tuple, TypeVar, Union, overload)"""))
+
```

Comment:
Which change actually fixed the parentheses-import syntax? It's not obvious to me...

Reply:
There's two. One change was in #2 and included a case where `import_as_names` begins with `(`. The second makes sure those changes are also utilized by the logic that handles `typing` imports. You can find a comment I made for the relevant code below